source/about/faq.rst

```diff
@@ -38,12 +38,12 @@ The Sponge Project currently develops two implementations which both use the Spo
     in. Sponge allows server owners to deploy Sponge plugins with ease, making server management easier. (SpongeForge
     was formerly known as Sponge, until it was renamed to avoid confusion).
 
-(2) **SpongeVanilla**, a stand-alone implementation of the Sponge API, running on top of the vanilla Minecraft server.
+(2) **SpongeVanilla**, a stand-alone implementation of the SpongeAPI, running on top of the vanilla Minecraft server.
```

Comment:
SpongeAPI is a name and should not have "the" in front of it. You wouldn't say "a SpongeAPI", but you _might_ say "a Sponge API". Indeed, "SpongeAPI is a Sponge API" - Sponge being the organisation that owns or is custodian of an API.

Thus:

> **SpongeVanilla**, a stand-alone implementation of SpongeAPI, running on top of the vanilla Minecraft server.

This comment applies all the way though.

Reply:
Indeed. From the issue DannyPS linked, I said "SpongeAPI is the proper noun denoting the actual API that SpongePowered make". There was further discussion [here](https://github.com/SpongePowered/SpongeDocs/issues/437#issuecomment-328901973) which is where this issue originally took root.